src/rules/no-extra-semicolons/__tests__/index.js

```diff
@@ -0,0 +1,463 @@
+import testRule from "../../../testUtils/testRule"
+
+import rule, { ruleName, messages } from ".."
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: "",
+    description: "empty string",
```

Comment:
Empty string is already covered by the basic checks that are run by default, I believe.